packages/sdk/src/waku/waku.ts

```diff
@@ -212,23 +220,9 @@ export class WakuNode implements IWaku {
     return this.connectionManager.isConnected();
   }
 
-  /**
-   * Return the local multiaddr with peer id on which libp2p is listening.
-   *
-   * @throws if libp2p is not listening on localhost.
-   */
-  public getLocalMultiaddrWithID(): string {
```

Comment:
@danisharora099 I didn't notice this function being used anywhere. What was the reason for introducing it? Do we still need to keep it?

Reply:
it's quite an old public method introduced by Franck, not sure of the reason for introducing it. I'd think it can be helpful to find the listening multiaddr for your own node, as it is helpful to MA with ID for nwaku that we need to dial. Since js-waku isn't really dialed by a lot of nodes directly (and it's the other way around), I would think we can remove it

Reply:
removing then